packages/ui/src/components/Tabs/style.less

```diff
@@ -10,8 +10,13 @@
 @tabs-hover-color: @condo-global-color-black;
 @tabs-horizontal-padding: @condo-global-spacing-4 0 @condo-global-spacing-12;
 @tabs-bar-margin: 0 0 @condo-global-spacing-40 0;
+@extra-content-margin: @condo-global-spacing-40;
 
 .condo-tabs {
+  &-tab {
+    align-items: end;
```

Comment:
What's there purpose of that rule?

Reply:
I discussed with the design and this task is a temporary solution until the button in the layout is moved and I was asked to make the text tabs and buttons on the same height 

with that rule:
<img width="790" alt="Screenshot 2023-05-19 at 13 42 43" src="https://github.com/open-condo-software/condo/assets/93817911/51cf90dc-5b89-4265-8f40-7e9fdbd04dd9">

without  that rule:
<img width="791" alt="Screenshot 2023-05-19 at 13 44 36" src="https://github.com/open-condo-software/condo/assets/93817911/a2fae306-f2e9-4a95-bac3-25f8cad4740f">